torch/nn/modules/batchnorm.py

```diff
@@ -35,8 +36,8 @@ def reset_parameters(self):
             self.running_mean.zero_()
             self.running_var.fill_(1)
         if self.affine:
-            self.weight.data.uniform_()
-            self.bias.data.zero_()
+            init.Ones(self.weight)
+            init.Zeros(self.bias)
```

Comment:
1. This changes the default initialization
2. Why aren't you using the public API?